code/controller/astar.cpp

```diff
@@ -227,6 +227,7 @@ void nrg::search_path(
     node *node_start = world[start.x()][start.y()];
     node *node_dest = world[dest.x()][dest.y()];
     astar_search_path(node_start, node_dest, world, node_path);
+    node_path.push_front(node_start);
```

Comment:
leave this out for now; we don't need the ObjectProcedure wasting time trying to get to the starting position